templates/_helpers.tpl

```diff
@@ -94,3 +94,36 @@ Create the name of the stackstorm-ha service account to use
 {{- end -}}
 {{- end -}}
     
+{{- define "init-containers-wait-for-db" -}}
+{{- if index .Values "mongodb" "enabled" }}
```

Comment:
Just thought about that edge case. Perfect to have it already codified here :+1: